cluster/canary/config_merger.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: testgrid-config-merger
+  namespace: testgrid-canary
+  labels:
+    app: testgrid
+    channel: stable
+    component: config-merger
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: testgrid
+      channel: stable
+      component: config-merger
+  template:
+    metadata:
+      labels:
+        app: testgrid
+        channel: stable
+        component: config-merger
+    spec:
+      serviceAccountName: config-merger
+      containers:
+      - name: config-merger
+        image: gcr.io/k8s-testgrid/config_merger:v20210309-v0.0.54-12-g4289ea4
+        args:
+        - --config-url="https://raw.githubusercontent.com/kubernetes/test-infra/master/testgrid/mergelists/canary.yaml"
```

Comment:
Note: I moved this, so the URL will be different. Will fix after other PR merges.